submissions/acerpo/index.html

```diff
@@ -0,0 +1,66 @@
+                            <a class="popup_content_body_link" href="#"><img src="./img/contacts.png" alt="Contacts">Contacts</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/tanslation.png" alt="Translate">Translate</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/videos.png" alt="Videos">Videos</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/search.png" alt="Search">Search</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/photos.png" alt="Photo">Photo</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/apps.png" alt="Apps">Apps</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/documents.png" alt="Document">Document</a>
+                            <div class="more">
+                                <input class="checkbox_input checkbox_more" type="checkbox" id="checkbox_toggle_more">
+                                <label class="more_label" for="checkbox_toggle_more">More</label>
+                                <div class="more_content">
+                                    <div class="more_content_body">
+                                        <a class="popup_content_body_link" href="#"><img src="./img/chat.png" alt="Chat">Chat</a>
+                                        <a class="popup_content_body_link" href="#"><img src="./img/classroom.png" alt="Classroom">Classroom</a>               
+                                        <a class="popup_content_body_link" href="#"><img src="./img/collection.png" alt="Collection">Collection</a>
+                                        <a class="popup_content_body_link" href="#"><img src="./img/mars.png" alt="Mars">Mars</a>
+                                        <a class="popup_content_body_link" href="#"><img src="./img/blogs.png" alt="Blogs">Blogs</a>
+                                        <a class="popup_content_body_link" href="#"><img src="./img/promo.png" alt="Promos">Promos</a>
+                                    </div>
+                                    <a class="more_footer" href="https://github.com/kottans/frontend">Even more from Kottans</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div> 
+                <div class="nav__item">
+                    <a class="headerLink" href="#"><img class="nav__item_img" src="./img/bell.png" alt="bell"></a>
+                </div>  
+                <div class="nav__item">
+                    <a class="headerLink" href="#"><img class=" user_img" src="./img/favicon.png" alt="user"></a>
+                </div>      
+            </nav>
+        </div>
+    </header>
+  
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)